tools/install-source/Makefile

```diff
@@ -49,11 +49,11 @@ MAC_MDB_FILES = \
 
 install-source-ios: install-source.exe
 	@echo "Installing source files for Xamarin.iOS"
-	$(Q) $(SYSTEM_MONO) install-source.exe $(IOS_ASSEMBLIES) --link:$(USE_SOURCE_LINKS) --mono-path=$(abspath $(MONO_PATH)) --opentk-path=$(abspath $(OPENTK_PATH)/Source) --xamarin-path=$(abspath $(SRC_PATH)) --install-dir=$(abspath $(IOS_DESTDIR))$(abspath $(MONOTOUCH_PREFIX))
+	$(Q) $(SYSTEM_MONO) install-source.exe $(IOS_ASSEMBLIES) --link:$(USE_SOURCE_LINKS) --mono-path=$(abspath $(MONO_PATH)) --opentk-path=$(abspath $(OPENTK_PATH)/Source) --xamarin-path=$(abspath $(SRC_PATH)) --install-dir=$(IOS_FRAMEWORK_DIR)/Versions/$(IOS_PACKAGE_VERSION)
```

Comment:
What's the reason for this change? It looks like it will do something to your system's XI instead of the locally built one when building normally.

Reply:
is not the case. Since we have the pathmap working on the build, we need to know the installation path to mangle the paths and find the correct source. Will not touch anything in the system, is just avoiding to hardcode anything in the manglers.